src/mission_execution/CMakeLists.txt

```diff
@@ -17,11 +17,17 @@ find_package(nav2_tasks REQUIRED)
 
 include_directories(
   include
+  /usr/local/include/BTpp
```

Comment:
Does the BTpp library not have a standard Ubuntu package? If not, I'm not sure how to specify the dependency.